src/ApplicationCore/Interfaces/IFraudPreventionService.cs

```diff
@@ -17,7 +20,9 @@ public interface IFraudProtectionService
     {
```

Comment:
What's the purpose of the region surrounding this interface?

Reply:
Same with the implementations

Reply:
The idea is that it would be easier for a dev to find all DFP specific code (vs sample site code) by searching for these regions.